build_manifest.yml

```diff
@@ -162,3 +162,9 @@ docs:
     - ^.*.nr$
   dependencies:
     - yarn-project
+
+noir:
+  buildDir: noir
+  arch:
```

Comment:
Is this used somewhere?

Reply:
Actually no. Well spotted. Intention is to replace some of the hardcoded stuff in config.yml with a manifest query. e.g.
```
create_ecr_manifest aztec-sandbox x86_64,arm64
create_ecr_manifest cli x86_64,arm64
```           
becomes
```
create_ecr_manifest aztec-sandbox
create_ecr_manifest cli
```

Reply:
And ultimately I'm still hopeful of fully code generating config.yml from the build manifest.